app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def self.update_comments_counter(post)
+    counter = Comment.where("post_id = #{post.id}").count
+    post.comments_counter = counter
+    post.save
+  end
```

Comment:
- [ ] [Optional] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the #update_post_counter method every time a new post has been saved to database, then make the update_post_counter method private